packages/mail/templates/cf_comment_notification_new.html

```diff
@@ -13,41 +13,51 @@
     </xml>
     <![endif]-->
     <style type="text/css">
-      @font-face {font-family:"GT-America-Standard-Regular";        src:url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-regular.eot);        src:url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-regular.eot?#iefix) format("embedded-opentype"),url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-regular.woff) format("woff"),url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-regular.ttf) format("truetype");}
-      @font-face {font-family:"GT-America-Standard-Medium";        src:url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-medium.eot);        src:url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-medium.eot?#iefix) format("embedded-opentype"),url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-medium.woff) format("woff"),url(https://cdn.republik.space/s3/republik-assets/fonts/gt-america-standard-medium.ttf) format("truetype");}
-      @font-face {font-family:Rubis-Regular;        src:url(https://cdn.republik.space/s3/republik-assets/fonts/rubis-regular.eot);        src:url(https://cdn.republik.space/s3/republik-assets/fonts/rubis-regular.eot?#iefix) format("embedded-opentype"),url(https://cdn.republik.space/s3/republik-assets/fonts/rubis-regular.woff2) format("woff2"),url(https://cdn.republik.space/s3/republik-assets/fonts/rubis-regular.woff) format("woff"),url(https://cdn.republik.space/s3/republik-assets/fonts/rubis-regular.ttf) format("truetype");}
+      *|SG_FONT_FACES|*
+    </style>
+    <!-- em and strong css for comment *|CONTENT|* (needs to be in head) -->
+    <style type="text/css">
+      em, i {
+        *|SG_FONT_STYLE_SERIF_ITALIC|*
```

Comment:
There is not example of `SG_FONT_STYLES`.

Wondering here if it may generate invalid HTML/CSS here if there is either of those styles undefined.

Thus: Since set of font styles seems limited and quite generic, maybe code itself should become aware of combinations and, in case either of it is missing, fill in nothing or a fallback.